external/kafka-assembly/pom.xml

```diff
@@ -66,13 +66,16 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>com.sun.jersey</groupId>
-      <artifactId>jersey-server</artifactId>
-      <scope>provided</scope>
+      <groupId>org.glassfish.jersey.core</groupId>
+      <artifactId>jersey-client</artifactId>
```

Comment:
Because it's already packaged with Spark. And this assembly will run inside of Spark. So there's no need to package it again.